examples/datadog/main.tf

```diff
@@ -89,6 +89,20 @@ module "k8s_platform" {
     GithubOrg   = "tx-pts-dai"
   }
 
+  karpenter = {
+    pod_annotations = {
```

Comment:
Would be good to bake this into the default karpenter release values and include if datadog is enabled otherwise this is required on the infra side.